pkg/karmadactl/cmdinit/utils/util_test.go

```diff
@@ -24,3 +25,11 @@ func TestDownloadFile(t *testing.T) {
 		panic(err)
 	}
 }
+
+// TestGetInitVersion test GetInitVersion
+func TestGetInitVersion(t *testing.T) {
```

Comment:
make it failable, so it could be useful in CI

Reply:
oh my fault.

Reply:
oh my fault.

Reply:
> make it failable, so it could be useful in CI
The version number is passed in when compiling, I feel like I can't do anything in go test.
https://github.com/karmada-io/karmada/blob/master/Makefile#L15

Reply:
`GetInitVersion()` get "GitVersion" from param, at least we could still check that the shift logic is correct or not